Makefile

```diff
@@ -27,16 +27,16 @@ proto-export-deps:
 PROTO_DIRS=$(shell find .proto-export-deps -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
 
 v4-proto-py-gen: proto-export-deps
-	@rm -rf ./v4-proto-py/v4-proto-py
-	@mkdir -p ./v4-proto-py/v4-proto-py
+	@rm -rf ./v4-proto-py/v4_proto
+	@mkdir -p ./v4-proto-py/v4_proto
 	@for dir in $(PROTO_DIRS); do \
 		python3 -m grpc_tools.protoc \
 		-I .proto-export-deps \
-		--python_out=./v4-proto-py/v4-proto-py \
-		--pyi_out=./v4-proto-py/v4-proto-py \
-		--grpc_python_out=./v4-proto-py/v4-proto-py \
+		--python_out=./v4-proto-py/v4_proto \
+		--pyi_out=./v4-proto-py/v4_proto \
+		--grpc_python_out=./v4-proto-py/v4_proto \
 		$$(find ./$${dir} -type f -name '*.proto'); \
 	done; \
-	touch v4-proto-py/v4-proto-py/__init__.py
+	echo "import os\nimport sys\n\nsys.path.insert(0, os.path.abspath(os.path.dirname(__file__)))" > v4-proto-py/v4_proto/__init__.py
 
```

Comment:
I don't like modifying the python path in here.

Reply:
I think it would be sufficient to just do `touch v4-proto-py/v4_proto/__init__.py` here?

Reply:
Resolved: this is a hack to resolve https://github.com/protocolbuffers/protobuf/issues/1491#issuecomment-429735834.

We should consider using sed to change the import statements instead. Mostly because we are hiding some real googleapi protos if we do this.